### DISCUSSION TITLE
[Unstable] Bug Report Periscope Base not switching to Periscope with turret uprade

### DISCUSSION THREAD

**Discussion by KonigP86** [FakeFishGames/Barotrauma#15240] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When purchasing a weapon for an empty turret hardpoint the associated periscope base is not switching to a periscope. It's happening on vanilla and custom subs

### Reproduction steps

1. Start game in outpost
2. Purchase new turret from Chief Mechanic
3. Leave outpost
4. Question how you will fire a turret from a Periscope base instead of a persicope.

First noticed on a custom sub, but was able to replicate on Dugong 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.7.4.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
